### DISCUSSION TITLE
Privacy Policy

### DISCUSSION THREAD

**Discussion by johndoe432** [d2phap/ImageGlass#1267] [Q&A] [answered]
Does this app collect any sorts of data besides the one required by the check-for-update feature? For how long collected data is stored?

**Comment by d2phap** [comment, marked as answer]
You can find the current privacy policy here: https://imageglass.org/privacy.

- ImageGlass anonymously appends a query string (includes the app's version) when you open a link from the app to https://imageglass.org.
- ImageGlass anonymously downloads a xml file from https://imageglass.org/checkforupdate to check for the new update, stores in its ConfigDir\Temp\ folder, and deletes after the app is closed.
